ui/packages/components/src/RunsPage/RunsPage.tsx

```diff
@@ -250,101 +256,108 @@ export function RunsPage({
 
   return (
     <main
-      className="bg-canvasBase text-basis h-full min-h-0 overflow-y-auto"
+      className="bg-canvasBase text-basis flex-1 overflow-auto"
       onScroll={onScroll}
       ref={containerRef}
     >
-      <div className="bg-canvasBase sticky top-0 z-10 flex items-center justify-between gap-2 px-4 py-2">
-        <div className="flex items-center gap-2">
-          <SelectGroup>
-            <TimeFieldFilter selectedTimeField={timeField} onTimeFieldChange={onTimeFieldChange} />
-            <TimeFilter
-              daysAgoMax={features.history}
-              onDaysChange={onDaysChange}
-              defaultValue={
-                lastDays
-                  ? {
-                      type: 'relative',
-                      duration: parseDuration(lastDays),
-                    }
-                  : startTime && endTime
-                  ? {
-                      type: 'absolute',
-                      start: new Date(startTime),
-                      end: new Date(endTime),
-                    }
-                  : {
-                      type: 'relative',
-                      duration: parseDuration(DEFAULT_TIME),
-                    }
-              }
+      <div className="bg-canvasBase z-5 border-subtle sticky top-0 flex flex-col border-b px-3">
```

Comment:
**question**: This is really good, but can we also make the table column headers sticky as well? I would think that in most experiences, the table headers should be fixed so a user as they scroll can tell which column is for start time, end time, etc.

Also in the video, I don't see the bottom border here showing - the filter bar kind of bleeds into the table rows

Reply:
The videos were recorded before I replaced the border-bottom! I'l take another look later to make the table header sticky